docs/modern/PaginationContainer.md

```diff
@@ -85,7 +85,7 @@ class Feed extends React.Component {
           edge => <Story story={edge.node} key={edge.node.id} />
         )}
         <button
-          onPress={this._loadMore}
+          onPress={() => this._loadMore()}
```

Comment:
Should not. 
Cause `this._loadMore` does not autobind `Feed` instance context for getting its props.